samples/moby-epub2-exploded/META-INF/com.apple.ibooks.display-options.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<display_options>
+    <platform name="*">
+        <option name="specified-fonts">true</option>
+        <option name="interactive">true</option>
+        <option name="fixed-layout">true</option>
+        <option name="open-to-spread">true</option>
+    </platform>
+    <platform name="iphone">
+        <option name="orientation-lock">landscape-only</option>
+    </platform>
+</display_options>
```

Comment:
This file doesn't do anything normally expect for the layout so I am just sticking it to our existing moby-epub sample

Reply:
Moving this to the `FXL` sample folder as suggested.